resources/views/includes/question/questions.blade.php

```diff
@@ -0,0 +1,33 @@
+<br></br>
+<div class="card">
+  <div class="card-header">{{ $church_name }}</div>
+
+  <div class="card-body">
+    <form action="{{ route('ratings.store') }}" method="POST">
+      @csrf
+
+      @foreach ($questions as $question)
+        <div class="form-group row">
+          <label for="  {{ $question->id }} " class="col-md-4 col-form-label text-md-right">{{ __($question->description ) }}</label>
+          <div class="col-md-6">
+            <input class="radio-inline m-2" type="radio" name="{{$question->id}}" value="1" required>1
+            <input class="radio-inline m-2" type="radio" name="{{$question->id}}" value="2">2
+            <input class="radio-inline m-2" type="radio" name="{{$question->id}}" value="3">3
+            <input class="radio-inline m-2" type="radio" name="{{$question->id}}" value="4">4
+            <input class="radio-inline m-2" type="radio" name="{{$question->id}}" value="5">5
```

Comment:
Over time we may want to make this dynamic so that it is easier to support different scales. This is fine for now.